### PR TITLE
Entities improvements

### PR DESCRIPTION
**Pull request by edwardGr**
Mapped entities to specific tables and columns according to project's database schema. Also added dataset for testing.

### CODE REVIEW

File: src/main/java/com/george/messenger/domain/dao/entity/ConversationEntity.java [hunk 1]
```diff
@@ -5,16 +5,26 @@
...
+            joinColumns = @JoinColumn(name = "conversation_id"),
+            inverseJoinColumns = @JoinColumn(name = "message_id")
+    )
     private Set<MessageEntity> messages;
 
     @ManyToMany
+    @JoinTable(name = "conversations_participants",
+        joinColumns = @JoinColumn(name = "conversation_id"),
+            inverseJoinColumns = @JoinColumn(name = "user_id")
+    )
```
Comment (VadymPolyanski) [inline src/main/java/com/george/messenger/domain/dao/entity/ConversationEntity.java]: good job

File: src/main/resources/schema.sql [hunk 2]
```diff
@@ -0,0 +1,75 @@
+CREATE TABLE conversations (
+  conversation_id BIGINT NOT NULL,
+  PRIMARY KEY (conversation_id)
+)
+  ENGINE = InnoDB;
```
Comment (VadymPolyanski) [inline src/main/resources/schema.sql]: what is that?

File: src/main/resources/test-dataset.sql [hunk 3]
```diff
@@ -0,0 +1,32 @@
+INSERT INTO
+  users
+  (user_id, age, first_name, gender, last_name, password, username)
+VALUES
+  ('513bb7b8-46d1-4a5c-90b4-207ad75e8bbb', 36, 'Jason', 0, 'Bourne', 'ciaagent123', 'jasonBourne'),
+  ('aa6c757e-5a5d-4276-9160-f9f6ef56dba3', 35, 'Irene', 1, 'Adler', 'irene123', 'ireneAdler');
+
+INSERT INTO
```
Comment (VadymPolyanski) [inline src/main/resources/test-dataset.sql]: ?

File: src/main/resources/schema.sql (hunk 2 above)
Comment (edwardGr) [inline src/main/resources/schema.sql]: I've extracted  database schema from production DB to use the same in tests DB. Am I wrong?

File: src/main/resources/test-dataset.sql (hunk 3 above)
Comment (edwardGr) [inline src/main/resources/test-dataset.sql]: Wrote some test data population sql. No the best practice as I see.

File: src/main/resources/schema.sql (hunk 2 above)
Comment (VadymPolyanski) [inline src/main/resources/schema.sql]: hibernate.hbm2ddl.auto=update means that your tables'll be built according to entity
but if you want another way, you can try liquibase(recomended)

Comment (edwardGr) [inline src/main/resources/schema.sql]: Ok. Got it.

File: src/main/resources/test-dataset.sql (hunk 3 above)
Comment (VadymPolyanski) [inline src/main/resources/test-dataset.sql]: where is your test package?
create junit tests for your repository (@RunWith(SpringJUnit4ClassRunner.class))
I use org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest annotation for dao tests but I don't know how it works with spring project(my project's based on spring boot)
If it doesn't work, you should create your own test jpa config (with h2 test db)

Comment (VadymPolyanski) [inline src/main/resources/test-dataset.sql]: I think dao tests isn't the primary task..

Comment (edwardGr) [inline src/main/resources/test-dataset.sql]: Okay. I ll focus on service layer. Thanks.

Comment (VadymPolyanski) [inline src/main/resources/test-dataset.sql]: remove redundant sql files and I'll merge this branch

Comment (edwardGr) [inline src/main/resources/test-dataset.sql]: Done.

Comment (VadymPolyanski) [inline src/main/resources/test-dataset.sql]: ?

Comment (edwardGr) [inline src/main/resources/test-dataset.sql]: I've deleted schema.sql and test-dataset.sql files.

Comment (VadymPolyanski) [inline src/main/resources/test-dataset.sql]: are you sure?

Comment (edwardGr) [inline src/main/resources/test-dataset.sql]: Sorry, deleted them on different branch.

**Pull request merged**